docs/03-tutorials/00-application-connectivity/ac-09-get-api-specification.md

```diff
@@ -6,13 +6,13 @@ To view or download the API specification directly from this website, see the **
 
 ## Connector Service API
 
-To get the API specification for the Connector Service on a local Kyma deployment, run this command:
+To get the API specification for Connector Service on a local Kyma deployment, run this command:
 
 ```bash
-curl https://connector-service.kyma.local/v1/api.yaml
+curl https://connector-service.local.kyma.dev/v1/api.yaml
```

Comment:
-k is missing. Consider using env variable and having one command.

Reply:
Correct me if I'm wrong, but I think the other command is to show that you don't need to curl it to get the spec. You can also paste it in your browser.

Why is the `-k` needed here? 
The command works for me as is:
![image](https://user-images.githubusercontent.com/53601578/142237636-d123b65e-ee5d-4475-8c92-90c5d20cfc14.png)

Reply:
My bad, I had the certificate stored locally so I did not need to pass the -k flag. Fixed.
Added a CAUTION note to add this flag when curling or to store a cert locally.